.gitignore

```diff
@@ -3,11 +3,15 @@ coverage
 config.yaml
 node_modules
 npm-debug.log
+package-json.lock
```

Comment:
It's still no bueno

Reply:
That got reintroduced as part of the the rebase. Fixed